stories/Form/Select.stories.mdx

```diff
@@ -8,8 +8,6 @@ import { Select } from '../../src';
 
 ### Il classico “menu a tendina”, in nuove varianti.
 
-La Select nelle sue varianti è basato su una libreria di terze parti, [react-select](https://react-select.com/).
-
 L'array di input usato per il componente `Select` in questo e per i successivi esempi è il seguente:
```

Comment:
Remove all this section.
Options are no longer js object, rather `option` element directly, so just render them explicitly in the story.